java-extensions/hadoop-ext/src/main/java/org/apache/hadoop/fs/HadoopExt.java

```diff
@@ -0,0 +1,43 @@
+package org.apache.hadoop.fs;
+
+import org.apache.hadoop.conf.Configuration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class HadoopExt {
+    private static final Logger LOGGER =
+            LoggerFactory.getLogger(HadoopExt.class);
+
+    public static final String HDFS_CONFIG_RESOURCES = "hadoop.config.resources";
+    public static final String HDFS_CONFIG_RESOURCES_LOADED = "hadoop.config.resources.loaded";
+    public static final String HDFS_RUNTIME_JARS = "hadoop.runtime.jars";
+    public static final String HDFS_CLOUD_CONFIGURATION_STRING = "hadoop.cloud.configuration.string";
+    public static final String STARROCKS_HOME_ENV = "STARROCKS_HOME";
+    public static final String LOGGER_MESSAGE_PREFIX = "[hadoop-ext]";
+
+    public static void addConfigResourcesToConfiguration(Configuration conf) {
+        if (conf.getBoolean(HDFS_CONFIG_RESOURCES_LOADED, false)) {
+            return;
+        }
+        String configResources = conf.get(HDFS_CONFIG_RESOURCES);
+        if (configResources == null || configResources.isEmpty()) {
```

Comment:
```java
String configResources = conf.getOrDefault(HDFS_CONFIG_RESOURCES, "");
if (configResources.isEmpty()) {}
```
better?

Reply:
I'm not sure. Maybe config resource is null in it.

Reply:
<img width="762" alt="image" src="https://github.com/StarRocks/starrocks/assets/18729228/a7625fd9-6320-4557-a47b-eaca940a0ffb">

conf not allow to insert null value